src/EmailCardList/EmailCard/EmailCard.js

```diff
@@ -1,7 +1,16 @@
-const EmailCard = (props) => {
+
+import './DisplayEmailCards.css'
+
+function EmailCard(props) {
+    const handleClick = event => {
+        props.setEmailSearchId(event.currentTarget.id)
+        props.setReadingPanelDisplay(true)
+    }
+  
     return (
-        <div className={(props.read == true ? 'btn-light' : 'btn-secondary') + " btn col-md-12 border-bottom d-flex flex-row justify-content-between p-2"}>
-            <div className="emailContainer">
+        <div className={(props.read == true ? 'btn btn-light col-md-12' : 'btn btn-secondary col-md-12') + "border d-flex flex-row justify-content-between p-2"} id={props.id}
```

Comment:
a previous problem has reemerged - in this ternary, the only class that is different between the true and false cases is `btn-light` vs `btn-secondary`, so the other classes need to be taken out of the ternary

Reply:
also rename `id` to `emailId` so that it is clear that it's a meaningful prop rather than an id attribute in an HTML sense